addons/mail/static/src/model/model_field_command.js

```diff
@@ -9,7 +9,7 @@ odoo.define('mail/static/src/model/model_field_command.js', function (require) {
 class FieldCommand {
     /**
      * @constructor
-     * @param {function} func function to call when executing this command
+     * @param {function} func function to call when executing this command. The function should ALWAYS return a boolean value to indicate if the field value is changed or not
```

Comment:
Also don't forget to add a dot at the end of the phrase.
And I would just reformulate "if the field value is changed or not" -> "whether the value changed."

Reply:
fixed